js/year_carte.js

```diff
@@ -1414,9 +1414,11 @@ $(function () {
     });
 
     // Switch the year from the dropdown
-    $('#year-select-AllPrenom').change(function () {
+    $('#autocomplete-input').change(function(){
+      setTimeout(function () {
```

Comment:
Here I added the timeout because I noticed that the autocomplete value was not updated right away.
E.g: if I've written 'Clau', and then clicked on 'Claudine', the value `val` was still 'Clau' instead of 'Claudine' for a short moment.
A few milliseconds after, it's solved :) hence the 500ms timeout.